### PR TITLE
One way associations

### PR DESCRIPTION
**Pull request by samselikoff**

### CODE REVIEW

**Comment by samselikoff** [comment]
Would love for people to test this out soon, I'll write up some of the changes needed.

Basically you need to define inverses (if they are defined), similar to how Ember Data works. Other than that things should work.

**Comment by knownasilya** [comment]
I'll test it once I have an example to follow 👍

**Comment by samselikoff** [comment]
@knownasilya here ya go! https://ember-twiddle.com/ec3a4c625c43e7a38f3c6c0c1b8232ec?openFiles=mirage.scenarios.default.js%2C

**Comment by dustinfarris** [comment]
Hey this is really great!

Dropping into my GraphQL app Just Continued to Work

`myManyRelationship.add(` is a welcome addition

Doing this in my scenarios/default.js works as well!:

```js
export default function(server) {
  server.createList('project', 2, {
    todos: server.createList('todo', 3)
  });
}
```

Nice work @samselikoff 👍

**Comment by samselikoff** [comment]
Wow that's great news! I'm writing a blog post now with more information.

One piece that's not in there right now is updating foreign keys after a delete, so you'll need to handle that logic yourself at the moment. I'm going to work it in soon though.

**Comment by samselikoff** [comment]
Dependents are now cleaned up after deletions (i.e. foreign key consistency)

File: addon/orm/schema.js [hunk 1]
```diff
@@ -202,6 +218,25 @@ export default class Schema {
...
+  }
+
+  dependentAssociationsFor(modelName) {
+    return this._dependentAssociations[modelName];
+  }
+
+  associationsFor(modelName) {
+    let modelClass = this.modelClassFor(modelName);
+
+    return Object.assign({}, modelClass.belongsToAssociations, modelClass.hasManyAssociations);
```
Comment (jelhan) [inline addon/orm/schema.js]: This breaks tests in phantomjs 2.1.1 since `Object.assign()` is not supported yet. I guess lodash assign should be used.

**Comment by jelhan** [comment]
@samselikoff I'm using 0.3.0-beta in a new project and it's working fine. Would be nice if you could release a beta6 dropping requirement of `Object.assign()`.

**Comment by jelhan** [comment]
@HeroicEric Thanks for fixing!

**Comment by samselikoff** [comment]
Will cut new beta release soon

**Pull request merged**